### ISSUE TITLE
How to use it without Vagrant

### ISSUE THREAD

**Issue by pydubreucq** [yhbyun/laravel-bookmark#8]
Hi,
I would like to know if it's possible to use laravel-bookmark without Vagrant ?
I would like to use it like a website and install all dependencies by hand.
Best Regards
Pierre-Yves

**Comment by yhbyun** [comment]
Yes, you can. There is no document for it, but I think it will be easy.

**Comment by pydubreucq** [comment]
Ok Thanks.
I will try and will do a little howto
Best Regards

2015-09-24 9:39 GMT+02:00 YongHun Byun <notifications@github.com>:

**Comment by cherepo** [comment]
Hi pydubreucq, have you figured out how to do it?

**Comment by pydubreucq** [comment]
Hi @cherepo 
Sorry I don't take the time now.
Best Regards

**Comment by stueja** [comment]
Also I would be interested.

Starting a general rant, not exclusively targeted to you, yhbyun:
I am looking for a simple, however good-looking bookmark application based on PHP and MySQL. 

Imagination: copy scripts, adjust web server configuration if necessary, go.

Outcome: found bookmark applications. However seemingly not intended for use on a normal nginx/php/mysql stack. Obviously forced to download a sh1teload of VMs, frameworks, nodejs, vagrant, composer, brew (again, rant not exclusively targeted to this project), programs to use VMs, frameworks, nodejs, vagrant, composer, brew, applications to download/configure/make VMs, frameworks, nodejs, vagrant, composer, brew. All cramming my system and making maintenance, updating and changing difficult. All cramming my brain with a steep learning curve for stuff which I might use once and never again. Why use x steps to install where 1 step of copying would be enough? Why is there php code which does not work without unmentioned and unknown-to-me prerequisites?

I probably go ahead and program my own, simple php and css.

Sorry for the rant,  /me needed venting after hours and hours of search. I could as well have written this into any of the other projects I found.

However, it shows only one of many perspectives. Other perspectives look more like 
thank you, yhbyun, for programming and maintaining this piece of creative work and making it available to the public. I am sure that there are many web developers who can use your project instantly. I am just a user, unwilling to take a steep, one-time-and-forget learning curve.

Note: I have walked through the steps of installing. Currently, running composer update, after the previous steps took 30% off my 15GB partition. Around 5GB for a couple php scripts. Hilarious.

Note2: http://192.168.22.10.xip.io/ -- This site can’t be reached; 192.168.22.10.xip.io took too long to respond. Needed to ip link set vboxnet0 up, ip addr 192.168.22.5 dev vboxnet0, ip route add 192.168.22.10 dev vboxnet0.

Note3:
root@homestead:/vagrant# du -hcs .
193M    .
193M    total
